backend/core/socketio_config.py

```diff
@@ -191,6 +194,17 @@ def handle_incoming_message(session: Session, message: dict):
     )
     session.add(new_chat)
     session.commit()
+
+    # handle media
+    if media:
+        for md in media:
+            new_media = Chat_Media(
+                chat_id=new_chat.id, url=md.get("url"), type=md.get("type")
+            )
+            session.add(new_media)
+            session.commit()
+    # eol handle media
+
```

Comment:
This code look very similar to the code from the Twilio client. Why this duplication, please?

Reply:
This is to save the media from client-to-user and the above one is from user-to-client.

Reply:
Yes, that already there. This function called inside client->user and user->client routing functions.

Reply:
See the comment about `add_media` in previous comment